utils/object_detection_utils.py

```diff
@@ -0,0 +1,296 @@
+###################################################################################################
+#
+# Copyright (C) 2022 Maxim Integrated Products, Inc. All Rights Reserved.
+#
+# Maxim Integrated Products, Inc. Default Copyright Notice:
+# https://www.maximintegrated.com/en/aboutus/legal/copyrights.html
+#
+###################################################################################################
+""" Some utility functions for Object Detection Tasks """
+
+import numpy as np
+import torch
+
+# GitHub repo for the below helper methods:
+# https://github.com/sgrvinod/a-PyTorch-Tutorial-to-Object-Detection:
+# MIT License
+# Copyright (c) 2019 Sagar Vinodababu

```

Comment:
Please move this copyright to the top header, and add ours. See train.py for an example.

Reply:
Done and commit will be amended